extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -82,9 +82,9 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.isSandbox_ = false;
 
     /**
-     * @private {Object<string, RequestHandler>} A map of request handler with requests
+     * @private {Object<!RequestHandler>} A map of request handler with requests
```

Comment:
But `this.request_` is still object<string, RequestHandler>

Reply:
yes, this is the shorthard for a map, by default key is string

Reply:
I see. Never know that : ) Thanks for explanation.